reconcile/utils/ocm.py

```diff
@@ -579,6 +582,28 @@ def delete_machine_pool(self, cluster, spec):
             f'{machine_pool_id}'
         self._delete(api)
 
+    def get_available_upgrades(self, version, channel):
+        """Get available versions to upgrade from specified version
+        in the specified channel
+
+        Args:
+            version (string): OpenShift version ID
+            channel (string): Upgrade channel
+
+        Raises:
+            KeyError: if specified channel is not valid
+
+        Returns:
+            list: available versions to upgrade to
+        """
+        if channel not in UPGRADE_CHANNELS:
+            raise KeyError(f'channel should be one of {UPGRADE_CHANNELS}')
+        version_id = f'openshift-v{version}'
+        if channel != 'stable':
+            version_id = f'{version_id}-{channel}'
+        api = f'{CS_API_BASE}/v1/versions/{version_id}'
+        return self._get_json(api).get('available_upgrades', [])
```

Comment:
The caller assumes these versions will always be sorted. Can you specify here or in the docstring if that is guaranteed to always be the case?

Reply:
i've changed the caller to use `reversed(sorted(available_upgrades))`, so there is no assumption.